summerofcode/2023.md

```diff
@@ -141,3 +141,34 @@ If you are a project maintainer and consider mentoring during the GSoC 2023 cycl
 - Difficulty: Medium
 - Upstream Issue (URL): https://github.com/strimzi/strimzi-kafka-operator/issues/8030
 
+### Curve
+
+#### Support metadata of Curve Block Storage stored in the database
+
+- Description: Currently the [metadata](https://github.com/opencurve/curve/blob/master/src/mds/nameserver2/namespace_storage.cpp) of current CurveBS is stored in [Etcd](https://github.com/opencurve/curve/tree/master/src/kvstorageclient). In order to apply to more scenarios, it is hoped that the metadata can add a sql database like mysql, MariaDB, PostgreSQL, etc. as one of the storage engines. Add a new sqlstorageclient for [metadata](https://github.com/opencurve/curve/blob/master/src/mds/nameserver2/namespace_storage.cpp).
+- Expected outcome: It is hoped that the user can choose whether the metadata is stored in the kv engine or the sql engine through the configuration file.
```

Comment:
I would have the same comments for the "Expected Outcome" section. What would this project help with?